package/Dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM alpine:3.9
-RUN apk add ebtables iptables
+FROM alpine:3.12 as network-controller
+# ebtables v2.0.10-4
+# iptables v1.6.2
+RUN echo -e 'http://dl-cdn.alpinelinux.org/alpine/v3.9/main' > /etc/apk/repositories && apk update && apk add ebtables iptables
```

Comment:
can u elaborate on the comment of why need to tag specific version here also per quote from the CI:
```
Use the `--no-cache` switch to avoid the need to use `--update` and remove `/var/cache/apk/*` when done installing packages (DL3019)
```